lib/pengine/pe_output.c

```diff
@@ -450,6 +458,28 @@ cluster_summary(pcmk__output_t *out, va_list args) {
                      scheduler->localhost, last_written, user, client, origin);
     }
 
+    // Use the existing scheduler, but avoid scheduler output
+    pcmk__output_new(&verify_out, "none", NULL, NULL);
+    //scheduler = pe_new_working_set();
+    priv_orig = scheduler->priv;
+    scheduler->priv = verify_out;
+
+    verify_rc = pcmk__verify(scheduler, verify_out, scheduler->input);
```

Comment:
What's happening is that `pcmk__verify()` calls `pcmk__schedule_actions()`, and then crm_simulate calls `pcmk__schedule_actions()` again, and it's not happy with being called twice.

One of these two approaches should work:

* Call `pe_reset_working_set()` after running `pcmk__verify()`. That will free `scheduler->input`, so you'll have to make a copy of that first and set it back afterward.
* Create a new scheduler object and copy the `input` and `now` members.